client/src/components/Video.js

```diff
@@ -0,0 +1,112 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+
+const Video = ({
+  id,
+  title,
+  url,
+  ratingUp,
+  ratingDown,
+  getAllVideos,
+  getDeleteMessage,
+}) => {
+  const videoId = url.split("v=")[1];
+
+  const deleteHandler = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_SERVERURL}/${id}`, {
+        method: "DELETE",
+      });
+      if (response.status !== 200) {
+        throw new Error("Something went wrong!");
+      }
+      const data = await response.json();
+      getDeleteMessage(data.message);
+      getAllVideos();
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const voteUpHandler = async () => {
+    try {
+      const response = await fetch(
+        `${process.env.REACT_APP_SERVERURL}/ratingup/${id}`,
+        {
+          method: "PUT",
+        }
+      );
+      if (response.status !== 200) {
+        throw new Error("Something went wrong!");
+      }
+      getAllVideos();
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const voteDownHandler = async () => {
```

Comment:
This looks very similar to the `voteUpHandler` - can you work out how to share their code in one function, rather than duplicate it?